internal/datastore/postgres/watch.go

```diff
@@ -160,7 +184,7 @@ func (pgd *pgDatastore) getNewRevisions(ctx context.Context, afterTX postgresRev
 	return ids, nil
 }
 
-func (pgd *pgDatastore) loadChanges(ctx context.Context, revisions []revisionWithXid) ([]datastore.RevisionChanges, error) {
+func (pgd *pgDatastore) loadChanges(ctx context.Context, revisions []revisionWithXid, options datastore.WatchOptions) ([]datastore.RevisionChanges, error) {
 	min := revisions[0].tx.Uint64
 	max := revisions[0].tx.Uint64
```

Comment:
collides with the new min/max functions

Reply:
This wasn't changed, but I changed it anyway